### PR TITLE
Fix 2 mistakes with reading analysisPeriod item

### PR DESCRIPTION
**Pull request by saeranv**
Hey @chriswmackey, this is a one line edit fixing a bug with the runUWG component's str(analysisPeriod)[x] inputs. 

For some reason the .py src file is saying there's 66 additions and 66 deletions, because it's not recognizing the existing code and counting it as new deletions/additions. Just ignore that, this is the only change I made: 
https://github.com/saeranv/Dragonfly/commit/7a83dd5fd11fe844f71998ab936bb9927f8ef768#diff-1c398502f6766062cf35516df227dbdbR383

Basically, two bugs fixed, first one is that you were checking str(analysisPeriod[0]) twice, and secondly, str(analysisPeriod[x]) contains spaces btwn list items, so you have to strip them for it to read correctly. Tested it and it reads it correctly now. 

S

### CODE REVIEW

**Comment by chriswmackey** [comment]
@saeranv ,
Thanks for fixing this!  Merged.
